apps/site/src/content/blog/state-of-the-budgie-2024.mdx

```diff
@@ -0,0 +1,273 @@
+
+Of course, it wouldn't be a month for me without some work being done on budgie-daemon v2. I continued work on the upcoming new Display Configuration Batch system, which is intended to completely overhaul the configuration logic to enable many queued operations to be consolidated and performed at once. This isn't just important from an architectural point-of-view but a usability one for end users. To describe why this matters, below is an excerpt on my [Today's Dev #15](https://joshuastrobl.com/todays-dev/15) where I talk about it:
+
+>  I want you to imagine your typical display configuration graphical software. You have rectangles, typically with labels in them describing the displays. At first, this is based off the current state of those heads (their position, resolution, refresh rate, adaptive sync, enabled / disabled, etc.). If you want to change the resolution and position of your monitors all in one go, you'd probably set their resolution so the rectangles change size, then move them around, then click apply. You're not changing the resolution, clicking apply, moving one monitor, clicking apply, moving another monitor, clicking apply.
+
+It is this sort of user experience that the system is intended to support.
+
+With Evan and David now testing out the daemon, I spent some time squashing some bugs they identified. Still more to be done on that front related to built-in displays.
+
+---
+
+### 2024 In Retrospect
+
+As you can see, there was a massive amount of work that happened over the course of 2024 towards our goal of moving to Wayland. As you may have noticed, we did not end up delivering Budgie 10.10 before end of year. As is the classic case of software development, some aspects of the experience have been proven to be trickier than anticipated, others requiring more work, some originally unscoped / undefined items needing to be worked on (labwc bridge as an example) to improve usability, and more.
+
+If we put it all into perspective though, we accomplished the massive goal of Budgie Desktop being daily-drivable on Wayland for a typical end user usecase. You are able to:
+
+- Log into a Budgie Desktop session on top of labwc from your login greeter of your choice
+- Launch applications and they're grouped as you would expect
+- Open up Budgie Menu, Raven, various popovers and they are correctly positioned.
+- Switch up your panel position.
+- Assuming you have the wlr and gtk xdg-desktop-portals installed, share your screen in applications.
+- Livestream using OBS Studio with screen capture working as expected
+- Once the labwc bridge PR lands, change many settings in Budgie Control Center and have them change the behavior of the compositor.
```

Comment:
Some items end in a period, while others do not. It would be nice to have them consistent.

Reply:
Should be resolved in ed01480